### ISSUE TITLE
low acc about svm with c3d

### ISSUE THREAD

**Issue by songhanjian** [facebook/C3D#379]
HI,dutran，I read your paper, you said use c3d+svm get a better performance, so I tried, first I use ucf101 set and conv3d_ucf101_train.prototxt(the original one in you c3d_train_ucf101 dir) to train and use the trained model to exact fc6 feature to retrain with linear svm,but I got a worse result which was only 0.10 acc,even use train set to test,i got only 0.66 acc meanwhile the result directly from softmax was 0.45 in test set and 0.998 in train set, I thought it was the problem came from net structure and train dataset, so I use your conv3d_deepnetA_sport1m_iter_1900000 to exact fc6,fc7 feature to go into linear svm,this time I tried fc6,fc7,and fc6,fc7 three ways, it is better than the last, but the result is also not good,i got only 0.17 acc,higher 5%,do you know why? is that sth wrong?and all feature before going into svm I use l2 normalization

**Comment by dutran** [comment]
It may be due to the cudnn supports, can you try the earlier one be for we merge the PR https://github.com/facebook/C3D/pull/327

**Comment by songhanjian** [comment]
will this effect ? but I didn't use cudnn when I install c3d,i think I will try next, there is another question,that in your paper, you exact fc6 with 8 frame overlap and then average them to a 4096 dimensions exactor after l2 norm, is that means the train.lst and test.lst has became 1 video 1 label instead of 16 frames 1 label, if it's true,than it's hard to compare cause test baseline has changed

**Comment by dutran** [comment]
Not sure I understand you correctly, but what I did is to extract feature for clips which 8-frame overlapped. Since a video has multiple clips, we average pool these clips features, L2-norm the averaged vector to make video representation. As a result, video has one label, then use linear SVM to train and test.

**Comment by DragotFly** [comment]
@dutran, I have the same problem with @songhanjian. I am using the Maryland dataset to test the C3D+SVM. I used C3D v1.0 (GPU implementation) to extract the clip features and can confirm the configuration is correct. I used LIBLINEAR as the linear SVM. For each video, I obtained a feature representation as you said:  " video has multiple clips, we average pool these clips features, L2-norm the averaged vector to make video representation." But for this dataset, I only have 130 videos, or 130 such 4096 dimensional video representations for training or testing. No matter how I tested the parameters of SVM, the accuracy is very low, <20%. It seems the training data is too few. Any suggestions?
